packages/react-components/react-dialog/bundle-size/Dialog.fixture.js

```diff
@@ -0,0 +1,8 @@
+import { Dialog } from '@fluentui/react-dialog';
+
+// eslint-disable-next-line no-console
+console.log(Dialog);
+
+export default {
+  name: 'Dialog (including children components)',
```

Comment:
There are no children components in the `Dialog` atm.

```suggestion
  name: 'Dialog',
```